packages/rum/src/viewCollection.ts

```diff
@@ -126,9 +126,11 @@ function newView(
     scheduleViewUpdate()
   })
 
-  const { stop: stopLoadingTimeTracking } = trackLoadingTime(lifeCycle, loadingType, (loadingTimeValue) => {
-    loadingTime = loadingTimeValue
-    scheduleViewUpdate()
+  const { stop: stopActivityLoadingTimeTracking } = trackActivityLoadingTime(lifeCycle, (newLoadingTime) => {
+    if (newLoadingTime !== undefined) {
+      updateLoadingTime(newLoadingTime)
```

Comment:
So for initial view, we could send a wrong loading time from activity and then correct it with the load event.
I would prefer to avoid that if possible

Reply:
Done